doc/dataframe.md

```diff
@@ -93,6 +93,40 @@ The keys in Redis:
 2) "person:Peter"
 ```
 
+The keys will not be persisted in Redis hashes
```

Comment:
I think we should also update 'Reading Redis Hashes' and 'DataFrame options' sections